### PR TITLE
Bugfix: Super Taki Amount

### PR DESCRIPTION
**Pull request by omerk2511**

### CODE REVIEW

File: source/cards/deck.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def build_initial_deck():
     return (reduce(lambda x, y: x + y,
                    [build_color_cards(color) for color in COLORS]) +
             [Card(CardType.CHANGE_COLOR, '', '')] * 2 +
```
Comment (Nisitay) [inline source/cards/deck.py:22]: According to the rules, there should be 4 change_color cards

**Review by Nisitay — APPROVED** [review]

File: source/cards/deck.py (hunk 1 above)
Comment (omerk2511) [reply source/cards/deck.py:22]: It is later multiplied by 2, which in turn makes it 4

Comment (Nisitay) [reply source/cards/deck.py:22]: Yeah I noticed that after I commented, got confused by the brackets

Comment (omerk2511) [reply source/cards/deck.py:22]: Legit

**Pull request merged**